tools/rosconsole/src/rosconsole/rosconsole.cpp

```diff
@@ -425,6 +436,19 @@ void initialize()
     backend::function_notifyLoggerLevelsChanged = notifyLoggerLevelsChanged;
     backend::function_print = _print;
 
+    std::string line_buffered;
+    if (get_environment_variable(line_buffered, "ROSCONSOLE_STDOUT_LINE_BUFFERED"))
+    {
+      if (line_buffered == "1") {
+        g_force_stdout_line_buffered = true;
+      } else {
+        if (line_buffered != "0") {
```

Comment:
Great, I changed it to `else if` in 52d967164773e8dee550bd96d15fb31aab0c3873 (plus some style adjustment to match the rest of the file since I was already in the editor :wink:). Once CI has passed again this can be merged.